src/redux/slices/meetingSlice.js

```diff
@@ -248,6 +286,18 @@ const meetingSlice = createSlice({
         state.isLoading = false;
         state.error = action.payload.message;
       })
+      .addCase(getActiveMeeting.fulfilled, (state, action) => {
+        state.meeting = action.payload;
+        state.isLoading = false;
+        state.error = '';
+      })
+      .addCase(getActiveMeeting.pending, (state) => {
+        state.isLoading = true;
+      })
+      .addCase(getActiveMeeting.rejected, (state, action) => {
+        state.isLoading = false;
+        state.error = action.error.message;
```

Comment:
change all of these error definitions on the rejected cases to `action.payload.response.data`